Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN git clone https://github.com/nvm-sh/nvm.git "$NVM_DIR" \
 
 COPY ./dist/cncjs $BUILD_DIR/cncjs
 COPY ./entrypoint $BUILD_DIR/cncjs/
+COPY ./index.hbs $BUILD_DIR/cncjs/server/views
```

Comment:
You don't need to copy the file. The file will be automatically copied to the `server/views` directory within the the [build-prod.sh](https://github.com/cncjs/cncjs/blob/master/scripts/build-prod.sh#L29) script.

Reply:
The `index.hbs` file, which serves as the HTML template, will be copied from the root directory to the output path `dist/cncjs/app` as defined in `webpack.config.production.js`:

```js
    new HtmlWebpackPlugin({
      filename: 'index.hbs',
      template: path.resolve(__dirname, 'index.hbs'),
      chunksSortMode: 'dependency' // Sort chunks by dependency
    })
```

![image](https://github.com/cncjs/cncjs/assets/447801/265609da-bf7b-4549-9569-7736998b906e)

Upon further reflection, I have come to understand that the build process for building a Docker image is not straightforward and is divided into several build scripts. Nevertheless, `index.hbs` has been removed and updated in the `next` branch.

Reply:
TL;DR:

I think you can leave the Dockerfile unchanged in this PR. Once the PR merged, I will trigger a CircleCI build to update existing Docker image with the tag 1.10.2 to fix this issue.